src/App.js

```diff
@@ -4,6 +4,7 @@ import 'bootstrap/dist/css/bootstrap.min.css';
 import 'antd/dist/antd.css';
 
 import Chart from './containers/Chart';
+import Pie from './containers/Pie';
```

Comment:
please `Pie => PieChart`, `Chart => StockChart`

Reply:
Sorry but I have components with names `PieChart` and `HighStockChart`. Could u please say how I must name components and containers?

Reply:
containers: PieChartContainer,
components: PieChart